man/preupgrade-assistant-api.1

```diff
@@ -110,11 +110,11 @@ The levels None, Slight and Medium change return code exit_fail to exit_needs_in
 
 There are several functions which do logging:
 
-\fBlog_{debug,info,warning,error} <component> <message>\fP
+\fBlog_{debug,info,warning,error} <message>\fP
 
 The function creates logs in the format:
 
-<SEVERITIES> <component> <TIMESTAMP> <MESSAGE>
+<SEVERITIES> <TIMESTAMP> <MESSAGE>
```

Comment:
This is actually not true, there's no timestamp anymore. The result is that the log messages are not shown in UI in the current state and will not be after this update.